schema.ts

```diff
@@ -96,10 +96,50 @@ export enum CurrencyFormat {
   Financial = 'financial',
 }
 
+export enum CurrencyCode {
```

Comment:
What happens if you use a currency code that's not in this list? Wondering if it works if we might just want to document that it's an ISO-whatever currency code rather than trying to keep these in sync as we add more to the product.

Reply:
We fall back to USD if they provide an unsupported currency code. I see, you think I should just add a comment saying that we fallback to USD if the currency is not supported?

Reply:
Yeah let's leave it as a string but add a doc comment that says something like:

```
/**
  A three-letter ISO 4217 currency code, e.g. USD or EUR. If the currency code is not supported by Coda, the value will be rendered using USD.
  */
```